nemo/collections/asr/models/transformer_bpe_models.py

```diff
@@ -599,9 +694,22 @@ def multi_validation_epoch_end(self, outputs, dataloader_idx: int = 0, eval_mode
                 sb_score = 0.0
                 wer_score = 0.0
 
-            self.log(f"{eval_mode}_loss", eval_loss, sync_dist=True)
-            self.log(f"{eval_mode}_sacreBLEU", sb_score, sync_dist=True)
-            self.log(f"{eval_mode}_WER", wer_score, sync_dist=True)
+            # To log via on_validation_epoch_end in modelPT.py
```

Comment:
@krishnacpuvvada see if we can remove logs and return dict to do what modelPT expects

CC @titu1994 curious how is it working currently